Cargo.toml

```diff
@@ -77,10 +75,70 @@ unused_qualifications = "warn"
 let_underscore_drop = "allow"
 
 [workspace.dependencies]
+# aries
 indy-vdr = { git = "https://github.com/hyperledger/indy-vdr.git", tag = "v0.4.3", default-features = false, features = [
     "log",
 ] }
 indy-vdr-proxy-client = { git = "https://github.com/hyperledger/indy-vdr.git", tag = "v0.4.3" }
 anoncreds = { git = "https://github.com/hyperledger/anoncreds-rs.git", tag = "v0.2.0" }
-aries-askar = { version = "0.3.1" }
-askar-crypto = { version = "0.3.1", default-features = false }
+anoncreds-clsignatures = "0.3.2"
+aries-askar = { version = "0.4.0" }
+askar-crypto = { version = "0.3.3", default-features = false }
+
+# common
+base64 = "0.22.1"
+bs58 = "0.5.1"
+multibase = "0.9.1"
+percent-encoding = "2"
+hex = "0.4.3"
+pem = "3.0.4"
+serde = { version = "1.0.217", default-features = false }
+serde_json = "1.0.134"
+derive_more = "2.0.1"
+async-trait = "0.1.83"
+futures = { version = "0.3.31", default-features = false }
+log = "0.4.22"
+url = { version = "2.5.4", default-features = false }
+uuid = { version = "1.11.0", default-features = false, features = ["v4"] }
+chrono = { version = "0.4.39", default-features = false }
+typed-builder = "0.20.0"
+thiserror = "2.0.11"
+anyhow = "1.0.95"
+regex = "1.11.1"
+lazy_static = "1.5.0"
+once_cell = "1.20.2"
+rand = "0.9.0"
+time = "0.3.37"
+bitvec = "1.0.1"
+sha2 = "0.10.8"
+unsigned-varint = "0.8.0"
+strum = "0.27.0"
+strum_macros = "0.27.0"
+syn = "2.0.98"
+quote = "1.0.38"
+clap = "4.5.6"
+
+# common drivers
+tokio = { version = "1.42.0", default-features = false }
+sqlx = "0.8.2"
+reqwest = "0.12.12"
+axum = "0.8.1"
+actix-web = "4"
+tower-http = "0.6.2"
+hyper= "1.6.0"
+hyper-tls = "0.6.0"
+hyper-util = "0.1.10"
+http-body-util = "0.1.2"
+native-tls = "0.2.12"
+
+# common dev
+mockall = "0.13.1"
+env_logger = "0.11.6"
+uniffi = "0.23.0"
+pretty_assertions = "1.4.0"
+dotenvy = "0.15"
+
+# TODO - await new indy-vdr release
```

Comment:
i've re-pinged indy-vdr discord, hopefully release is soon...

Reply:
TODO - switch to just use the indy-vdr commit on hyperledger repo and add a comment justifying.